src/scss/_base.scss

```diff
@@ -57,15 +57,14 @@ h1, h2, h3, h4, h5, h6 {
 
 .hc-footer-text-white {
     color: #ced4da;
-}
-
-.hc-footer-text-white-bigger {
-    letter-spacing: 0.08em;
-}
+    &-bigger {
+      letter-spacing: 0.08em;
+    }
+  }
 
-.hc-phone-num {
+  a[href^="tel:+14153214567"] {
```

Comment:
What if somebody uses another phone number somewhere?
A more generic one would be better.